### PR TITLE
Remove MSS badge and add ding to product thumbnails

### LINKED ISSUE #5425: PNI - product thumbnails - remove MSS badge and add dings
- [x] remove MSS badge from product thumbnails
- [x] show ding on product thumbnails if applicable

---
###  Design -> Dev Handoff Checklist
- [x] Design has been finalized
- [x] Link to related ticket: #5314
- [x] Link to Abstract (make sure it can be viewed by anyone)
- [x] Design changes been merged into the master Sketch file
  - Merged into master Sketch file via Abstract. (_Master > Buyers Guide.sketch > [Home Page Updates (Q4 2020](https://share.goabstract.com/07a9aca5-3903-4f00-828a-6ac8bebb5be3?sha=e49d77cc9da6a70e8c412cd985e28138a3091cd5))_).
- [x] Desktop and mobile mockups are included:
  - [[Desktop] Homepage - Index](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=103e293c-16c0-48af-bac2-dc7d7545e5ea&mode=design) 
  - [[Mobile] Homepage](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=3f690fff-c322-4f11-8505-04aee49fea04&mode=design) 
- [x] Name of the Sketch file (and specific page/section to look at, if applies)
- [x] Assets are included. Link: https://drive.google.com/open?id=1kfrvpO-yguTrPWkOjY3P2yiSvOxeuk1G&authuser=sabrina%40mozillafoundation.org&usp=drive_fs
- [x] Hover state for elements are designed: none

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #5425

https://foundation-s-issue-5425-tl8bej.herokuapp.com/privacynotincluded

### CODE REVIEW

**Review by sabrinang — APPROVED** [review]
👍

File: network-api/networkapi/buyersguide/templates/catalog.html [hunk 1]
```diff
@@ -45,13 +45,13 @@ <h1 class="h1-heading text-center mt-0">{% trans "Be Smart. Shop Safe." %}</h1>
...
-                  {% if product.votes.confidence.1 > product.votes.confidence.0 %}
-                      <div class="d-none recommendation positive"></div>
-                  {% else %}
-                      <div class="d-none recommendation negative"></div>
-                  {% endif %}
+              {% if product.privacy_ding %}
+              <img
+                src="{% static "_images/buyers-guide/icon-privacy-ding-mini.svg" %}"
+                title="{% trans "*privacy not included with this product" %}"
+                alt="{% trans " *privacy not included with this product" %}"
```
Comment (TheoChevalier) [inline network-api/networkapi/buyersguide/templates/catalog.html:52]: Removing this extra space will allow us to translate this once for both attributes

```suggestion
                alt="{% trans "*privacy not included with this product" %}"
```

**Review by TheoChevalier — CHANGES_REQUESTED** [review]

**Review by Pomax — APPROVED** [review]
r+ with theo's requested fix

File: network-api/networkapi/buyersguide/templates/catalog.html [hunk 2]
```diff
@@ -45,13 +45,13 @@ <h1 class="h1-heading text-center mt-0">{% trans "Be Smart. Shop Safe." %}</h1>
 
         <figure class="product-box d-flex flex-column justify-content-between{% if product.draft %} draft-product{% endif %}{% if product.adult_content %} adult-content{% endif %}" data-creepiness="{{ product.votes.creepiness.average }}">
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/catalog.html:46]:
```suggestion
        <figure class="product-box d-flex flex-column justify-content-between{% if product.draft %} draft-product{% endif %}{% if product.adult_content %} adult-content{% endif %}{% if product.privacy_ding %} privacy-ding{% endif%}" data-creepiness="{{ product.votes.creepiness.average }}">
```

**Review by Pomax — CHANGES_REQUESTED** [review]
[n/a]

**Review by Pomax — APPROVED** [review]

**Review by TheoChevalier — APPROVED** [review]
👍 Thanks!

**Comment by mmmavis** [comment]
I tried rebasing but it took forever to fix conflicts so I've created a new PR for this. See https://github.com/mozilla/foundation.mozilla.org/pull/5535.

Closing this one.

**Pull request closed without merging**